appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockComponent.java

```diff
@@ -155,8 +155,8 @@ public void onClick(ClickEvent event) {
         }
       });
       HorizontalPanel buttonPanel = new HorizontalPanel();
-      buttonPanel.add(cancelButton);
```

Comment:
For consistency, I think it is better for the Cancel to come before the OK/Delete buttons. There are other places in the UI where this occurs (create/delete project confirmation, even the browser dialog in Chrome). I think we should keep this original ordering where the cancel is first (left) and the ok is second (right). Otherwise, we should change all other dialogs as well, not just the two pertaining to MockComponents.